src/app/find/mapPage/filters/filters.css

```diff
@@ -29,3 +29,13 @@ input#hivNutrition-checkbox {
 	margin-left: 32px;
 	color: #333333;
 }
+
+.filterSubmitBtn {
+    position: fixed;
```

Comment:
One issue I've often seen with fixed-position bottom elements: If the contents of the screen take up 100vh or more, often the bottom-most elements can never be seen properly, since scrolling down all the way will leave the floating element covering them.
One way around that is to add sufficient bottom padding to the whole container.

Is this an issue here, or is there already sufficient empty space at the bottom to accommodate the button?

Reply:
Correct, the pb-5 should solve that. I tested it on multiple different screen sizes (including very old phones like Nexus 4 and iPhone 4) and noticed no problems so it should be sufficient.